src/Pim/Component/Catalog/Normalizer/Indexing/Product/NumberNormalizer.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Pim\Component\Catalog\Normalizer\Indexing\Product;
+
+use Pim\Component\Catalog\Model\ProductValueInterface;
+use Symfony\Component\Serializer\Normalizer\NormalizerInterface;
+
+/**
+ * Normalizer for a decimal or an integer product value
+ *
+ * @author    Samir Boulil <samir.boulil@akeneo.com>
+ * @author    Anaël Chardan <anael.chardan@akeneo.com>
+ * @author    Philippe Mossière <philippe.mossiere@akeneo.com>
+ * @copyright 2017 Akeneo SAS (http://www.akeneo.com)
+ * @license   http://opensource.org/licenses/osl-3.0.php Open Software License (OSL 3.0)
+ */
+class NumberNormalizer extends AbstractProductValueNormalizer implements NormalizerInterface
+{
+    const DECIMAL_PRECISION = 4;
```

Comment:
We take it from https://github.com/jjanvier/pim-community-dev/blob/TIP-613-unified/src/Pim/Component/Catalog/Normalizer/Standard/Product/ProductValueNormalizer.php